backend/src/main/resources/application.yaml

```diff
@@ -54,10 +54,14 @@ simple-report-initialization:
       manufacturer: LumiraDx UK Ltd.
       model: LumiraDx SARS-CoV-2 Ag Test*
       loinc-code: "95209-3"
-simple-report:
-  data-hub:
-    uploadurl: "https://prime-data-hub-test.azurefd.net/api/reports"
-    maxcsvrows: 999
 logging:
   pattern:
     console: "%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(%5p) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} [Query: %X{graphql-query}] %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n%wEx"
+simple-report:
+  data-hub:
+    # these are overridden by application-dev, -prod, -test, etc
+    upload-enabled: false
+    upload-url: ""
+    max-csv-rows: 999
+    api-key: ""
+    secret-slack-notify-webhook-url: ""
```

Comment:
We could pull the env-variable injection up into this file and save some repetition in the other ones.